CHANGELOG.md

```diff
@@ -1,10 +1,19 @@
 # Change Log
 
+## [0.6.1](https://github.com/openeuropa/oe_theme/tree/0.6.1) (2018-08-13)
+[Full Changelog](https://github.com/openeuropa/oe_theme/compare/0.6.0...0.6.1)
+
+**Merged pull requests:**
+
+- OPENEUROPA-772: Update compose.json to have openeuropa/drupal-core-require-dev. [\#118](https://github.com/openeuropa/oe_theme/pull/118) ([voidtek](https://github.com/voidtek))
+- OPENEUROPA-772: Move styleguide into require-dev. [\#117](https://github.com/openeuropa/oe_theme/pull/117) ([richardcanoe](https://github.com/richardcanoe))
```

Comment:
I would here explain why the style guide has been moved. For example:

> OPENEUROPA-772: The Style Guide module has been moved to a development dependency since it is not required for using the theme in production environments.